Makefile

```diff
@@ -242,19 +242,15 @@ cppcheck:
 		2>cppcheck.log
 	@echo "Done! See cppcheck.log for details."
 
-install-newsboat: $(NEWSBOAT) doc/$(NEWSBOAT).1
+install-newsboat: $(NEWSBOAT)
 	$(MKDIR) $(DESTDIR)$(prefix)/bin
 	$(INSTALL) $(NEWSBOAT) $(DESTDIR)$(prefix)/bin
-	$(MKDIR) $(DESTDIR)$(mandir)/man1
-	$(INSTALL) -m 644 doc/$(NEWSBOAT).1 $(DESTDIR)$(mandir)/man1 || true
 
-install-podboat: $(PODBOAT) doc/$(PODBOAT).1
+install-podboat: $(PODBOAT)
 	$(MKDIR) $(DESTDIR)$(prefix)/bin
 	$(INSTALL) $(PODBOAT) $(DESTDIR)$(prefix)/bin
-	$(MKDIR) $(DESTDIR)$(mandir)/man1
-	$(INSTALL) -m 644 doc/$(PODBOAT).1 $(DESTDIR)$(mandir)/man1 || true
 
-install-docs: doc
+install-docs: doc doc/$(NEWSBOAT).1 doc/$(PODBOAT).1
```

Comment:
These new dependencies are already present via the `doc` target [1].

Side note: At that place [1], `doc/newsboat.1` is specified verbatim instead of `doc/$(NEWSBOAT).1`.
Is tit useful to make that consistent?

[1] https://github.com/newsboat/newsboat/blob/fc29d0ab2d082c82168b1b039f8b3fb61e7675dd/Makefile#L173

Reply:
Both notes addressed. Thanks!